### ISSUE TITLE
Error occurred when building theme zip archive

### ISSUE THREAD

**Issue by 0xCyph3rr** [jimbobbennett/FauxGhost#1]
I cloned your repo and installed all stuff. Then I just tried to launch gulp default task without editing any file:

```
✗ gulp
[19:04:45] Using gulpfile ~/FauxGhost/FauxGhost/gulpfile.js
[19:04:45] Starting 'clean_dest'...
[19:04:45] Starting 'clean_release'...
[19:04:45] Starting 'clean_zip'...
[19:04:45] Finished 'clean_dest' after 9.71 ms
[19:04:45] Finished 'clean_release' after 4.64 ms
[19:04:45] Finished 'clean_zip' after 5.58 ms
[19:04:45] Starting 'scss'...
[19:04:45] 'scss' errored after 2.38 ms
[19:04:45] TypeError: Arguments to path.join must be strings
    at path.js:360:15
    at Array.filter (native)
    at Object.exports.join (path.js:358:36)
    at module.exports (/home/datenshi/FauxGhost/FauxGhost/node_modules/gulp-ruby-sass/index.js:65:15)
    at Gulp.gulp.task.eventstream.concat.gulp.src.pipe.pipe.pipe.pipe.rename.suffix (/home/datenshi/FauxGhost/FauxGhost/gulpfile.js:18:15)
    at module.exports (/home/datenshi/FauxGhost/FauxGhost/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/home/datenshi/FauxGhost/FauxGhost/node_modules/gulp/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/home/datenshi/FauxGhost/FauxGhost/node_modules/gulp/node_modules/orchestrator/index.js:214:10)
    at /home/datenshi/FauxGhost/FauxGhost/node_modules/gulp/node_modules/orchestrator/index.js:279:18
    at finish (/home/datenshi/FauxGhost/FauxGhost/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:21:8)
```

**Comment by 0xCyph3rr** [comment]
gulp-ruby-sass syntax need to be updated with 1.x version

Example:

```
// gulp-ruby-sass: 1.x
gulp.task('sass', function() {
    return sass('path/to/scss', { style: 'expanded' })
        .pipe(gulp.dest('path/to/css'));
});
```

This trick was found on http://stackoverflow.com/questions/28140012/gulp-typeerror-arguments-to-path-join-must-be-strings

**Comment by 8bitAlex** [comment]
I got the same. Have yet to dive in to the files to try and change it.

```
[23:25:12] Using gulpfile ~/Downloads/FauxGhost-master/gulpfile.js
[23:25:12] Starting 'clean_dest'...
[23:25:12] Starting 'clean_release'...
[23:25:12] Starting 'clean_zip'...
[23:25:12] Finished 'clean_dest' after 32 ms
[23:25:12] Finished 'clean_release' after 19 ms
[23:25:12] Finished 'clean_zip' after 14 ms
[23:25:12] Starting 'scss'...
[23:25:12] 'scss' errored after 4 ms
[23:25:12] TypeError: Arguments to path.join must be strings
    at Object.posix.join (path.js:471:13)
    at module.exports (/Users/hardline/Downloads/FauxGhost-master/node_modules/gulp-ruby-sass/index.js:65:15)
    at Gulp.<anonymous> (/Users/hardline/Downloads/FauxGhost-master/gulpfile.js:18:15)
    at module.exports (/Users/hardline/Downloads/FauxGhost-master/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/hardline/Downloads/FauxGhost-master/node_modules/gulp/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/hardline/Downloads/FauxGhost-master/node_modules/gulp/node_modules/orchestrator/index.js:214:10)
    at /Users/hardline/Downloads/FauxGhost-master/node_modules/gulp/node_modules/orchestrator/index.js:279:18
    at finish (/Users/hardline/Downloads/FauxGhost-master/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:21:8)
    at /Users/hardline/Downloads/FauxGhost-master/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:52:4
    at f (/Users/hardline/Downloads/FauxGhost-master/node_modules/gulp/node_modules/orchestrator/node_modules/end-of-stream/node_modules/once/once.js:17:25)
```

**Comment by jimbobbennett** [comment]
Sorry - I haven't touched this code in a while so it's not up to date.  Will try to get some time to resolve - unless you want to make the changes and raise a PR?

**Comment by jimbobbennett** [comment]
Sorry - I haven't touched this code in a while so it's not up to date.  Will try to get some time to resolve - unless you want to make the changes and raise a PR?

**Comment by kevinjalbert** [comment]
Should this issue be closed if #2 fixed it? (I didn't have any issues running `gulp`)
